### PR TITLE
Update beaglebone_blue.md

### PR DESCRIPTION
**Pull request by Superbear88**

### CODE REVIEW

File: en/flight_controller/beaglebone_blue.md [hunk 1]
```diff
@@ -57,7 +57,8 @@ connmanctl>scan wifi
 connmanctl>services
 #(at this point you should see your network SSID appear.)
 connmanctl>agent on
-connmanctl>connect <SSID>
+connmanctl>connect wifixxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
```
Comment (hamishwillee) [inline en/flight_controller/beaglebone_blue.md:60]: I am not sure about this. Unless you literally enter `wifixxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx`, then I would leave as it was, and after this maybe add a note:

> Note, the format of the `<SSID>` is normally the text 'wifi' followed by a string of other characters. After entering the command you will be prompted to enter the wifi password.

**Comment by Superbear88** [comment]
Note is good, cause SSID i test with the normal SSID, but you cant put the clear SSID, you ned the wifi... what you get with connmanctl services

File: en/flight_controller/beaglebone_blue.md (hunk 1 above)
Comment (hamishwillee) [inline en/flight_controller/beaglebone_blue.md:60]: Revert
```suggestion
connmanctl>connect <SSID>
```

File: en/flight_controller/beaglebone_blue.md [hunk 2]
```diff
@@ -58,6 +58,7 @@ connmanctl>services
 #(at this point you should see your network SSID appear.)
 connmanctl>agent on
 connmanctl>connect <SSID>
+	Enter Passphrase
 connmanctl>quit
 ```
 ### SSH root Login on Beaglebone
```
Comment (hamishwillee) [inline en/flight_controller/beaglebone_blue.md:64]:
```suggestion

:::note
The format of the `<SSID>` above is normally the text 'wifi' followed by a string of other characters.
After entering the command you will be prompted to enter the wifi password.
:::

### SSH root Login on Beaglebone
```

**Comment by hamishwillee** [comment]
Cool. I have updated it now - does it look OK to you @Superbear88 ?

**Comment by Superbear88** [comment]
Looks perfect.

**Pull request merged**